docs/application/web/tutorials/app-filtering.md

```diff
@@ -260,6 +261,7 @@ Tizen 4.0.
 | `http://tizen.org/feature/network.bluetooth.audio.media` | Specify this key, if the application requires the Bluetooth Advanced Audio feature (A2DP) feature for using the [Media Key](../api/latest/device_api/wearable/tizen/mediakey.html) API. | 2.3   |
 | `http://tizen.org/feature/network.bluetooth.health` | Specify this key, if the application requires the Bluetooth Health feature (HDP). | 2.3   |
 | `http://tizen.org/feature/network.bluetooth.le` | Specify this key, if the application requires the Bluetooth Low Energy feature (BLE). | 2.3   |
+| `http://tizen.org/feature/network.bluetooth.le.gatt.client` | Specify this key, if the application requires the Bluetooth Low Energy GATT Client feature. | 4.0   |
```

Comment:
Question. We also have "**Table: Available requirements for TV Web Device APIs**" table.
Don't we need to update that table to include `http://tizen.org/feature/network.bluetooth.le.gatt.client`?

Reply:
Thank you for this suggestion, I've added it to TV section also.